app/views/schools/toggle_enabled/edit.html.erb

```diff
@@ -1,4 +1,11 @@
-<% self.page_title = "Toggle requests" %>
+<% self.page_title = "Toggle requests"
```

Comment:
Question for @gemmadallmandfe - I think we should change the page title for this 'Toggle requests' feels poorly named and not all that descriptive. What do you think?

Reply:
Agree that we should match the title to the H1 @MylesJarvis